responsibleai/requirements.txt

```diff
@@ -1,6 +1,6 @@
 dice-ml>=0.6.1,<0.7
 econml==0.12.0b4
-erroranalysis>=0.1.8
+erroranalysis>=0.1.11
 interpret-community>=0.18.1
```

Comment:
Do we need to upgrade interpret-community to 0.19.x?

Reply:
I wanted to hold off on that since it's so new in case it causes any environment conflicts I can't foresee